lib/OpenQA/Utils.pm

```diff
@@ -88,15 +88,22 @@ sub needledir {
     return productdir($distri, $version) . '/needles';
 }
 
-sub needle_info($$$) {
+sub needle_info($$$$) {
```

Comment:
I think we don't need and want prototypes anymore so you actually can remove whole `($$$)` block.